tutorial/practicals/nemo/1_nemo_psyir/README.md

```diff
@@ -173,7 +195,7 @@ interesting from a performance point of view.
 To familiarise yourself with PSyIR navigation, you can:
```

Comment:
For line 116, in my previous PR we discussed the "." vs "source", but I didn't realise we had the ".csh" at the end. I would replace it with ".sh" as I assume everybody will be using bash or more modern.

Reply:
Done.